site/js/components/hero-animated.js

```diff
@@ -53,25 +65,56 @@ export class HeroAnimated {
 
     /** @type {AnimationConfig} */
     this.animationConfig_ = {
-      container: this.elem_.querySelector(this.constants_.wrapperSelector),
+      container: this.elem_.querySelector(this.constants_.selectors.wrapper),
       animType: 'svg',
       loop: true,
       animationData: animationData,
       autoplay: false,
-      name: this.constants_.animationName,
+      name: this.constants_.animation.name,
       rendererSettings: {
         progressiveLoad: true,
-        className: this.constants_.animationItemClass,
       },
     };
 
     this.init_();
   }
 
   /**
-   * Inits the animation and sets the event handlers.
+   * Check for user preferences and use the theme to apply the corresponding animation.
+   */
+  async useTheme() {
+    if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
+      const phosphor = document.body.dataset.theme === 'phosphor';
```

Comment:
The only place this is getting called (in the constructor in the event listener) passes in the theme name. Let's use that instead here